src/controllers/update/update.beneficial.owner.other.controller.ts

```diff
@@ -0,0 +1,141 @@
+import { NextFunction, Request, Response } from "express";
```

Comment:
lot's of code duplication in this controller from registration.

Reply:
Created the following [branch](https://github.com/companieshouse/overseas-entities-web/tree/poc/uar-260-implementation) for possible implementation and reference.